GCRCatalogs/dc2_object.py

```diff
@@ -296,22 +314,25 @@ def _generate_modifiers(pixel_scale=0.2, bands='ugrizy'):
                 modifiers['I{}_{}'.format(ax, band)] = '{}_base_SdssShape_{}'.format(band, ax)
                 modifiers['I{}PSF_{}'.format(ax, band)] = '{}_base_SdssShape_psf_{}'.format(band, ax)
 
-            modifiers['mag_{}_cModel'.format(band)] = (
-                lambda x: -2.5 * np.log10(x) + 27.0,
-                '{}_modelfit_CModel_flux'.format(band),
-            )
-
-            modifiers['magerr_{}_cModel'.format(band)] = (
-                lambda flux, err: (2.5 * err) / (flux * np.log(10)),
-                '{}_modelfit_CModel_flux'.format(band),
-                '{}_modelfit_CModel_fluxSigma'.format(band),
-            )
-
-            modifiers['snr_{}_cModel'.format(band)] = (
-                np.divide,
-                '{}_modelfit_CModel_flux'.format(band),
-                '{}_modelfit_CModel_fluxSigma'.format(band),
-            )
+            if version == '1.1':
+                modifiers['mag_{}_cModel'.format(band)] = (
+                    lambda x: -2.5 * np.log10(x) + 27.0,
+                    '{}_modelfit_CModel_flux'.format(band),
+                )
+                modifiers['magerr_{}_cModel'.format(band)] = (
+                    lambda flux, err: (2.5 * err) / (flux * np.log(10)),
+                    '{}_modelfit_CModel_flux'.format(band),
+                    '{}_modelfit_CModel_fluxSigma'.format(band),
+                )
+                modifiers['snr_{}_cModel'.format(band)] = (
+                    np.divide,
+                    '{}_modelfit_CModel_flux'.format(band),
+                    '{}_modelfit_CModel_fluxSigma'.format(band),
+                )
+            else:
+                modifiers['mag_{}_cModel'.format(band)] = '{}_modelfit_mag'.format(band)
+                modifiers['magerr_{}_cModel'.format(band)] = '{}_modelfit_mag_err'.format(band)
+                modifiers['snr_{}_cModel'.format(band)] = '{}_modelfit_SNR'.format(band)
```

Comment:
Move _current_ case outside conditional statement, so that in the future when removing deprecated code, it'll be a cleaner removal.

```python
            modifiers['mag_{}_cModel'.format(band)] = '{}_modelfit_mag'.format(band)
            modifiers['magerr_{}_cModel'.format(band)] = '{}_modelfit_mag_err'.format(band)
            modifiers['snr_{}_cModel'.format(band)] = '{}_modelfit_SNR'.format(band)

            if version == '1.1':
                modifiers['mag_{}_cModel'.format(band)] = (
                    lambda x: -2.5 * np.log10(x) + 27.0,
                    '{}_modelfit_CModel_flux'.format(band),
                )
                modifiers['magerr_{}_cModel'.format(band)] = (
                    lambda flux, err: (2.5 * err) / (flux * np.log(10)),
                    '{}_modelfit_CModel_flux'.format(band),
                    '{}_modelfit_CModel_fluxSigma'.format(band),
                )
                modifiers['snr_{}_cModel'.format(band)] = (
                    np.divide,
                    '{}_modelfit_CModel_flux'.format(band),
                    '{}_modelfit_CModel_fluxSigma'.format(band),
                )
```